modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -12803,6 +12803,7 @@ only-one-sku-can-be-approved=Only one SKU can be approved.
 only-one-value-is-allowed-in-single-selection-mode=Only one value is allowed in "Single" selection mode.
 only-required-in-this-asset-library=Only Required in This Asset Library
 only-show-results-for-web-content-listed-in-a-web-content-display-widget=Only show results for web content listed in a web content display widget.
+only-style-books-based-on-the-frontend-token-definition-provided-by-x-theme-are-visible=Only style books based on the frontend token definition provided by {0} theme are visible
```

Comment:
We probably have a period at the end of the sentence in the design. You should add it here as well